src/styles/components/buttons.less

```diff
@@ -35,6 +35,29 @@ components/buttons.less
         }
       }
     }
+
+    &-up,
```

Comment:
I'm not sure I like those `&-up`, `&-down`, `&-wide`, etc. It makes it hard to search and find your class definition